### PR TITLE
[ADDED] Cleanup Logging

### PR DESCRIPTION
**Pull request by petemiron**
Closes Issue #62 .

Changed prefix to STREAM from STAN.

Please double check unit test in server/log_test.go. It's a little weird due to the empty string case.

Also, to confirm, should we lowercase the first word in all log messages?

### CODE REVIEW

File: server/log_test.go [hunk 1]
```diff
@@ -147,8 +147,12 @@ func TestLogOutput(t *testing.T) {
 	d := &dummyLogger{}
 
 	checkLogger := func(output string) {
-		if d.msg != output {
-			t.Fatalf("Unexpected logger message: %v", d.msg)
+		checkMsg := output
```
Comment (kozlovic) [inline server/log_test.go]: Please revert those changes. You don't need to do anything special here. I am using a dummy logger (see code above) and this is not using the normal logger. So whatever text is passed to `Noticef`, `Debugf`, etc.. in this test is set in `dummyLogger.msg`.

**Comment by kozlovic** [comment]
Go says that if you create an `error` (even with `fmt.Errorf()`), the first letter should be lower case and not have period at the end. However, this is our logger and that does not apply. For instance, linters should not report those as warnings.

File: server/log_test.go (hunk 1 above)
Comment (kozlovic) [inline server/log_test.go]: After discussion, I realize my mistake since the addition of the prefix is done before delegating to the (dummy) logger.

**Comment by ColinSullivan1** [comment]
LGTM.

**Pull request merged**